ida-customizations/ida-course/LPs/week-0/index.md

```diff
@@ -21,7 +21,17 @@ If you do not need a blockchain refresher, feel free to start your journey with
 
 </HighlightBox>
 
-Working with Cosmos also means working with Golang. If you have not worked much with Go, do not worry. The introduction to Golang is meant as a useful overview of Go, and a starting point for your journey in the Interchain Ecosystem and developing with the Cosmos SDK.
+Do you want to refresh your blockchain knowledge? This chapter provides a general introduction to blockchain technology. Feel free to skip this chapter if you do not need a refresher and start your journey with the Week 1 content, which will be available by May 18th at 4 pm UTC when the program officially starts.
+
+Working with Cosmos also means working with Golang. If you have not worked much with Go, do not worry.
+
+<HighlightBox type="info">
+
+As all blockchain ecosystems keep on growing and maturing, product stacks and terminologies evolve. Thus, please be aware that Cosmos is also known as **the Interchain**. The terms "Cosmos", "Cosmos Ecosystem", and "Interchain" are synonymous with each other, and the various tools available to developers ("the stack") can be referred to collectively as the **Interchain Stack**.
+
+</HighlightBox>
+
+This introduction to Golang is meant as a useful overview of Go and as such, a starting point for your journey in the Cosmos Ecosystem and developing with the Cosmos SDK.
```

Comment:
@AndrewB9lab This is the only meaningful remaining difference in this PR. If you think it is not important, just close the PR.

Reply:
Yes, I think this is superseded by what we have now.